OpenOversight/app/main/views.py

```diff
@@ -1098,17 +1095,17 @@ def get_dept_ranks(department_id: int = 0, is_sworn_officer: bool = False):
 
 
 @main.route("/department/<int:department_id>/units")
-def redirect_get_dept_units(department_id: int = 0):
+def redirect_get_dept_units(department_id: int):
     flash(FLASH_MSG_PERMANENT_REDIRECT)
     return redirect(
-        url_for("main.get_dept_ranks", department_id=department_id),
+        url_for("main.get_dept_units", department_id=department_id),
         code=HTTPStatus.PERMANENT_REDIRECT,
     )
 
 
 @main.route("/departments/<int:department_id>/units")
 @main.route("/units")
-def get_dept_units(department_id: int = 0):
+def get_dept_units(department_id: Optional[int] = None):
```

Comment:
Could you add `methods` for this function and the redirect one?

Reply:
`app.route` methods defaults to `["GET"]`. Is there any reason we need to specify methods?

Reply:
I prefer minimal assumptions when possible, but it's not a deal breaker.